docs/src/modules/javascript/pages/views.adoc

```diff
@@ -38,88 +50,57 @@ include::example$js-customer-registry/customer_view.proto[tag=unary]
-Possible filter predicates are:
-
-* `=` equals
-* `!=` not equals
-* `>` greater than
-* `>=` greater than or equals
-* `<` less than
-* `\<=` less than or equals
+[#register-view]
+=== Register the View
 
-The filter conditions can be combined with `AND`/`OR`.
+In the View implementation, register the View with Akka Serverless. In addition to passing the service descriptor and a unique identifier, pass any descriptors that define state. In this example, the `customer_domain.proto` descriptor defines the Value Entity state:
 
-[source,proto,indent=0]
+.customer-value-entity-view.js
+[source,js,indent=0]
 ----
-SELECT * FROM customers WHERE
-  name = :customer_name AND address.city = 'New York' OR
-  name = :customer_name AND address.city = 'San Francisco'
+include::example$js-customer-registry/customer-value-entity-view.js[tag=register]
 ----
 
-[#register-view]
-== Registering the View
-
-Once you've defined a View, register it with the `AkkaServerless` server by invoking the `addComponent` function.
+To use the View in the client, invoke the `addComponent` function. For example:
 
 .index.js
 [source,js,indent=0]
 ----
 include::example$js-customer-registry/index.js[tag=register]
 ----
 
-.customer-value-entity-view.js
-[source,js,indent=0]
-----
-include::example$js-customer-registry/customer-value-entity-view.js[tag=register]
-----
-
-In addition to passing the service descriptor of the View, and a unique identifier of the View, you also need to pass any descriptors that you use for events, for example, the `domain.proto` descriptor.
-
 [#event-sourced-entity]
 == View from Event Sourced Entity
 
-The previous example derived a View from a Value Entity and used state changes to update the View. In contrast, to create a View from an Event Sourced Entity, you use events that the Entity emits to build a state representation.
+Construct Event Sourced Entity Views from the events that the Entity emits. Build a state representation from the events and Query them. Using a Customer Registry service example, to create a View for querying customers by name:
 
-=== Protobuf definition
+. <<_define_the_view_descriptor>>.
+. <<_implement_transformation_functions>>.
+. <<#ES_register>>.
 
-Like the Value Entity example above, this View will provide a way to query customers. The Protobuf file defines the following events that will update the View:
+
+The example assumes a `customer_domain.proto` file defines events that will update the View because a name changed:
 
 [source,proto,indent=0]
 ----
 include::example$js-customer-registry/customer_domain.proto[tag=events]
 ----
 
-The following lines in the `.proto` file define a View to consume the `CustomerCreated` and `CustomerNameChanged` events:
+=== Define the View descriptor
+
+A view descriptor:
+
+* Defines update methods for events.
+* Provides the source of the View.
+* Enables transformation updates.
+* Specifies a `table` attribute used by Akka Serverless to store the View. You can use any name.
```

Comment:
Again, could be useful to mention that the given table name is used to refer to this table in queries.

Reply:
Note, it's table `attribute` here, while it was table `property` for value entities. Could be useful to use the same term.

Reply:
fixed.

Reply:
fixed where?

Reply:
See https://github.com/lightbend/akkaserverless-javascript-sdk/pull/139/files